styles/globals.scss

```diff
@@ -161,9 +161,20 @@ body {
   margin: 0;
 }
 
+.WtePercentage {
+  display: inline-flex;
+  align-items: center;
+  column-gap: 2000px;
```

Comment:
That's some column gap! 
What does the column-gap do?

Reply:
yea I was just thinking the same, quite a few tiny commits